packages/react-composites/src/composites/ChatComposite/adapter/AzureCommunicationChatAdapter.ts

```diff
@@ -496,29 +495,33 @@ export class AzureCommunicationChatAdapter implements ChatAdapter {
   }
 }
 
-const convertEventToChatMessage = (event: ChatMessageReceivedEvent): ChatMessage => {
+const convertEventToChatMessage = (
+  event: ChatMessageReceivedEvent | ChatMessageEditedEvent | ChatMessageDeletedEvent
+): ChatMessage => {
   return {
     id: event.id,
     version: event.version,
-    content: { message: event.message },
+    content: isChatMessageDeletedEvent(event) ? undefined : { message: event.message },
     type: convertEventType(event.type),
     sender: event.sender,
     senderDisplayName: event.senderDisplayName,
     sequenceId: '',
-    createdOn: new Date(event.createdOn)
+    createdOn: new Date(event.createdOn),
+    editedOn: isChatMessageEditedEvent(event) ? event.editedOn : undefined,
```

Comment:
This property can be returned for `messageDeleted` event from ChatSDK and we probably would want to be consistent

Reply:
It's not in the chatClient event that we are forwarding here, only editedEvent has the editedOn:
![image](https://github.com/Azure/communication-ui-library/assets/2684369/3afe3abd-d0a7-46c2-98f5-d1d0950c97fd)